src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/NoOpRestoreUtilities.cs

```diff
@@ -23,13 +23,13 @@ internal class NoOpRestoreUtilities
         /// </summary>
         internal static bool IsNoOpSupported(RestoreRequest request)
         {
-            return request.DependencyGraphSpec != null && request.ProjectStyle != ProjectStyle.DotnetCliTool;
+            return request.DependencyGraphSpec != null;
         }
 
         /// <summary>
         /// The cache file path is $(BaseIntermediateOutputPath)\$(project).nuget.cache
         /// </summary>
-        internal static string GetPJorPRCacheFilePath(RestoreRequest request)
+        private static string GetPJorPRCacheFilePath(RestoreRequest request)
```

Comment:
Good idea on the naming. 
We have more project styles than restore styles (netcore, legacy vs only packagereference style), so I'd rather not duplicate the code.